.github/workflows/ci.yml

```diff
@@ -142,24 +145,74 @@ jobs:
           classpath-build-command: "mvn test-compile -ntp -Pags -pl \"-:alfresco-community-repo-docker\""
 
   all_unit_tests_suite:
-    name: "Core, Data-Model, Repository - AllUnitTestsSuite - Build and test"
+    name: ${{ matrix.testName }} - AllUnitTestsSuite - Build and test
     runs-on: ubuntu-latest
     needs: [prepare]
     if: >
       !contains(github.event.head_commit.message, '[skip repo]') &&
       !contains(github.event.head_commit.message, '[skip tests]') &&
       !contains(github.event.head_commit.message, '[force')
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - testName: Core
+            testModule: core
+            testAttributes: "-Dtest=AllCoreUnitTestSuite"
+          - testName: Data-Model
+            testModule: data-model
+            testAttributes: "-DskipCoreTests=true -Dtest=AllDataModelUnitTestSuite"
```

Comment:
Out of curiosity, why does this need the `-DskipCoreTests=true` while other elements in the matrix don't?